app/component/itinerary/TransitLeg.js

```diff
@@ -574,6 +584,7 @@ class TransitLeg extends React.Component {
           )}
           {routeNotifications}
           <LegAgencyInfo leg={leg} />
+          {children}
```

Comment:
Debugger shows a warning: 

Warning: Failed prop type: The prop `children` is marked as required in `TransitLeg`, but its value is `undefined`.

Please fix that. Also, while testing, I found another warning (an old bug):
[React Intl] Missing message: "bus-express-with-route-number" for locale: "fi", using default message as fallback. 
 
Could you add the missing messages into translations.js ?  A test search, which shows the translation error in detail view:
http://localhost:8080/reitti/Eira%2C%20Helsinki%3A%3A60.156706%2C24.937662/Alppiharju%2C%20Helsinki%3A%3A60.189746%2C24.947151/?time=1721278800